main/lsp/request_dispatch.cc

```diff
@@ -130,7 +130,11 @@ LSPResult LSPLoop::processRequestInternal(unique_ptr<core::GlobalState> gs, cons
             prodCategoryCounterInc("lsp.messages.processed", "initialize");
             auto &params = get<unique_ptr<InitializeParams>>(rawParams);
             if (auto rootUriString = get_if<string>(&params->rootUri)) {
-                rootUri = *rootUriString;
+                if(rootUriString->at(rootUriString->length() - 1) == '/') {
```

Comment:
```suggestion
                if(absl::EndsWith(rootUriString, "/")) {
```

We're using abseil to do common operations in a more readable way, so it's not really standard c++ :-)

Reply:
(this will also handle empty string, while your code crashes on it)

Reply:
I've made this change, thanks! :)